node/src/reactor/main_reactor/tests/binary_port.rs

```diff
@@ -495,6 +496,21 @@ fn chainspec_raw_bytes(network_chainspec_raw_bytes: ChainspecRawBytes) -> TestCa
     }
 }
 
+fn latest_switch_block_header() -> TestCase {
+    TestCase {
+        name: "latest_switch_block_header",
+        request: BinaryRequest::Get(GetRequest::Information {
+            info_type_tag: InformationRequestTag::LatestSwitchBlockHeader.into(),
+            key: vec![],
```

Comment:
This makes me think whether `key` should be an `Option<Vec<u8>>`? But definitely, not in scope of this PR.

Reply:
IMO this is OK, I think it might be worth renaming the `key` field to `key_bytes` though, because that's what it basically is